pallets/parachain-staking/src/lib.rs

```diff
@@ -507,6 +561,12 @@ pub mod pallet {
 	pub(crate) type CandidateInfo<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, CandidateMetadata<BalanceOf<T>>, OptionQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn candidate_last_active)]
+	/// Stores a counter that represents during how many rounds a collator has not produced blocks
+	pub(crate) type CandidateLastActive<T: Config> =
```

Comment:
```suggestion
	pub(crate) type CandidateInactiveSince<T: Config> =
```

The name `CandidateLastActive` suggest that the value is the last round at which the candidate produced a block, which is false.